book/team.yaml

```
@@ -355,6 +355,30 @@ people:
     user_groups:
       - Tutorial Lead
   # =============
+  - title: Shamsudeen Yekeen
+    avatar: 
```

To add your picture, change this line to:
```suggestion
    avatar: https://avatars.githubusercontent.com/u/44532329?v=4
```